XamarinCommunityToolkit/Helpers/RegularExpressions.shared.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace XamarinCommunityToolkit.Helpers
+{
+    public static class RegularExpressions
+    {
+        public static string Email => @"^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-0-9a-z]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$";
+        public static string PhoneNumber_US => @"/^\b\d{3}[-.]?\d{3}[-.]?\d{4}\b$/";
```

Comment:
This doesn't seem to be used, if that is the case, I would remove it for now

Reply:
This is actually used in the email validation in XAML:

![image](https://user-images.githubusercontent.com/2419439/84742252-80e7c800-afb0-11ea-9688-7e3c47859dd5.png)